inlang/source-code/website/src/server/vike-plugin.ts

```diff
@@ -0,0 +1,24 @@
+import express, { Router } from "express"
```

Comment:
let's consolidate all files in server into the main.ts file. it's quite unübersichtlich atm

Reply:
@samuelstroschein I put the code of `vike-plugin.ts` and `redirect.ts` inside the `router.ts`. I think we can keep the routing in `router.ts` or should this be consolidated to `main.ts` as well?